docs/development.md

```diff
@@ -116,3 +116,28 @@ You'll need to create a test repository and install your app by clicking the "In
 
 1. Always run `$ npm install` and restart the server if `package.json` has changed.
 1. To turn on verbose logging, start server by running: `$ LOG_LEVEL=trace npm start`
+
+## Alternate way of running a probot app
+
+If you take a look to the `npm start` script, this is what it runs: `probot run ./index.js`. This is nice, but you sometimes need more control on how your Node.js application is executed. For example if you want to use custom V8 flags, `ts-node`, etc. you need more flexibility. In those cases there's a simple way of executing your probot application programmatically:
+
+```js
+// main.js
+const { Probot } = require('probot')
+const app = require('./index.js')
+
+// pass a probot app as a function
+Probot.run(app)
```

Comment:
I wonder if it would make sense to pass in options here? I’m thinking of @Eddman’s usecase to have a custom `Octokit` constructor: https://github.com/probot/probot/pull/840

I’m not sure how it would work exactly, the code base is still confusing me 😜

Reply:
I don't know the code base a lot. Maybe we can implement this in a different PR?

Reply:
sounds good 👍